app/tests/unit/components/Form/ProjectFundingAgreementFormSummary.test.tsx

```diff
@@ -247,13 +248,11 @@ describe("The Project Funding Agreement Form Summary", () => {
 
     expect(screen.getByText("50.00 %")).toBeInTheDocument();
     expect(screen.getByText("60.00 %")).toBeInTheDocument();
-    expect(screen.getByText("$200.00")).toBeInTheDocument(); //max funding changes
     expect(screen.getByText("$300.00")).toBeInTheDocument(); // max funding changes
     expect(screen.getByText(/Jan[.]? 1, 2021/)).toBeInTheDocument();
-    expect(screen.getByText(/Feb[.]? 2, 2021/)).toBeInTheDocument();
   });
 
-  it("Displays all data for an EP projectwhen isFirstRevision is true (Project Creation)", () => {
+  it("Displays all data for an EP project when isFirstRevision is true (Project Creation)", () => {
     componentTestingHelper.loadQuery({
       ProjectRevision() {
```

Comment:
we can add `...mockQueryPayloadEP,` here and remove query below:
```
      Query() {
        return {
          epFundingParameterFormBySlug: {
            jsonSchema: projectFundingParameterEPSchema,
          },
          iaFundingParameterFormBySlug: {
            jsonSchema: projectFundingParameterIASchema,
          },
        };
      },
```

this way we don't need to repeat this query every time.